pkg/command/client.go

```diff
@@ -265,6 +266,211 @@ func (c *Client) updatePanelTargets(queryManager *QueryManager, panel *simplejso
 	return nil
 }
 
+func (c *Client) parseQueryPaths(panelDesc string) []string {
```

Comment:
Can we use the new helper [here](https://github.com/diogogmt/grafctl/blob/master/pkg/command/client.go#L211)?